x-pack/legacy/plugins/siem/public/components/fields_browser/index.test.tsx

```diff
@@ -116,6 +125,51 @@ describe('StatefulFieldsBrowser', () => {
         wrapper.find(`.field-browser-category-pane-auditd-${timelineId}`).first()
       ).toHaveStyleRule('font-weight', 'bold');
     });
+    test('it updates the selectedCategoryId state according to most fields returned', done => {
+      const wrapper = mount(
+        <TestProviders>
+          <StatefulFieldsBrowser
+            browserFields={mockBrowserFields}
+            columnHeaders={[]}
+            height={FIELD_BROWSER_HEIGHT}
+            onUpdateColumns={jest.fn()}
+            timelineId={timelineId}
+            toggleColumn={jest.fn()}
+            width={FIELD_BROWSER_WIDTH}
+          />
+        </TestProviders>
+      );
+
+      wrapper
+        .find('[data-test-subj="show-field-browser"]')
+        .first()
+        .simulate('click');
+
+      wrapper
+        .find(`.field-browser-category-pane-auditd-${timelineId}`)
+        .first()
+        .simulate('click');
+
+      wrapper.update();
+      expect(
+        wrapper.find(`.field-browser-category-pane-default.ECS-${timelineId}`).first()
+      ).toHaveStyleRule('font-weight', 'normal');
+      wrapper
+        .find('[data-test-subj="field-search"]')
+        .last()
+        .simulate('change', { target: { value: 'default' } });
+
+      setTimeout(() => {
+        wrapper.update();
+        expect(
+          wrapper.find(`.field-browser-category-pane-default.ECS-${timelineId}`).first()
+        ).toHaveStyleRule('font-weight', 'bold');
```

Comment:
Thanks for adding this test! Due to what appear to be DOM changes since the original code was written, the font-weight applied here doesn't show up visually, because the EUI text below the component is now wrapped in a `div`. One possible solution that works (in my local branch) is to update `CategoryName` in `x-pack/legacy/plugins/siem/public/components/fields_browser/category_columns.tsx` such that the style targets any child containing the `.euiText` class:

```
const CategoryName = styled.span<{ bold: boolean }>`
  .euiText {
    font-weight: ${({ bold }) => (bold ? 'bold' : 'normal')};
  }
`;
```